app/src/androidTest/java/org/mozilla/focus/activity/URLCompletionTest.java

```diff
@@ -67,4 +92,208 @@ public void CompletionTest() throws InterruptedException, UiObjectNotFoundExcept
         TestHelper.hint.waitForExists(waitingTime);
         assertTrue (TestHelper.inlineAutocompleteEditText.getText().equals("http://www.mozilla.org"));
     }
+
+    @Test
+    public void CustomCompletionTest() throws UiObjectNotFoundException {
```

Comment:
\[PMD\] INFO: The method 'CustomCompletionTest\(\)' has a NCSS line count of 20\.